exam_date/stored_date.py

```diff
@@ -0,0 +1,72 @@
+from autologging import logged, traced
+from pathlib import Path
+from datetime import datetime
+
+
+@logged
+@traced
+class AssignmentLatestSubmittedDate():
+    def __init__(self, path: str, file_name: str):
+        self.path = path
+        self.file_name = file_name
+
+    def _create_directory_if_not_exit(self) -> None:
```

Comment:
I notice you specify a return type of `None` here and other functions return other types.  However, you don't use the `typing` module, because it's not needed.  I mention this as an example of why it's probably not needed in the other file, "entry.py", either.

Reply:
Removed